src/app/services/StudentService.ts

```diff
@@ -0,0 +1,19 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+import type { StudentResponse } from "../types/responses/StudentResponse.type";
+
+export const StudentService = createApi({
+  reducerPath: "students",
+  baseQuery: fetchBaseQuery({
+    baseUrl: `${import.meta.env.VITE_BACKEND_URL}/students`,
+  }),
+  endpoints: (builder) => ({
+    deleteStudent: builder.mutation<StudentResponse, string>({
```

Comment:
La API, cuando realizas un `DELETE`, no devuelve ninguna respuesta, por lo que acá deberíamos decir que no se espera ningun resultado 


```suggestion
    deleteStudent: builder.mutation<void, string>({
```